src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILiteralOperation.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void LiteralFLow_01()
```

Comment:
>LiteralFLow_01 [](start = 20, length = 14)

Should we have a test for at least one other literal type here: say string? Don't think we need to go through the full set here.

Reply:
Ah just caught up on email which said this is sufficient. 

---
In reply to: [186608888](https://github.com/dotnet/roslyn/pull/26691#discussion_r186608888) [](ancestors = 186608888)